swift/.bazelversion

```diff
@@ -0,0 +1 @@
+6.0.0
```

Comment:
Why can't it use the repo's main .bazelversion, which is pinned to 7.0.0-pre.20221207.2? It doesn't work with Bazel 7?

Reply:
No, we are not using versiob 7 but version 5.2.0

Reply:
Wait, so what is https://github.com/EngFlow/engflow/blob/master/.bazelversion used for?

Reply:
Wait, this PR is for another repository, not engflow. This is example.

Reply:
You mean, this will be used as an example, so you want it to be self-contained?

Reply:
I don't know how to make it work with different version than the repo itself. So I added a workspace and its own .bazelversion. The repository .bazelversion is this: https://github.com/EngFlow/example/blob/main/.bazelversion

Reply:
Oh, sorry, I all this time didn't realize that this PR is on the example repo :-)
Now I'm wondering why the example repo is pinned to an old Bazel version...

Reply:
I was thinking on moving to version 6. WDYT?